docs/ruleset_functions/get_zone_target_baseline_system.md

```diff
@@ -0,0 +1,127 @@
+# get_zone_target_baseline_system
+**Schema Version:** 0.0.22  
+
+**Description:** Following G3.1.1, determines the baseline system type for each zone in a building
+
+**Inputs:**
+- **P-RMI**
+- **B-RMI**
+
+**Returns:**  
+- **zones_and_systems**: a dictionary with zone / list pairs where the first value in the list is the expected system type (ex SYS_3) and the second value is the rule used to choose the system, (eg "G3_1_1e"): zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] = SYS_3; zones_and_systems[zone]["SYSTEM_ORIGIN"] =  "G3_1_1e"
+ 
+**Function Call:**
+- **get_HVAC_building_area_types_and_zones**
+- **does_zone_meet_G3_1_1c**
+- **does_zone_meet_G3_1_1e**
+- **does_zone_meet_G3_1_1f**
+- **does_zone_meet_G3_1_1g**
+- **get_predominant_HVAC_building_area_type**
+- **get_number_of_floors**
+- **expected_system_type_from_Table_G3_1_1**
+
+**Data Lookup**
+- **Table G3.1.1**
+
+## Logic:
+- create zones_and_systems list: `zones_and_systems = {}`
+- pre-fill the list with an empty string:
+	- `for building_segment in RMR.building.building_segments:`
+		- `for zone in building_segment.zones:`
+			- `zones_and_systems[zone] = {}`
+- get the list of HVAC building area types and zones: `list_building_area_types_and_zones = get_HVAC_building_area_types_and_zones(B-RMR)`
+- get the predominant building area type: `predominant_building_area_type = get_predominant_HVAC_building_area_type(list_building_area_types_and_zones)`
+- get the total number of floors for the building: `num_floors = get_number_of_floors(B-RMI)`
+- first get the expected system type for the predominant building area type.  To do this, we need the total area of the predominant building area type, which includes all BAT's less than 20000 ft2
+- set area equal to the area of the predominant building area type: `area = list_building_area_types_and_zones[predominant_building_area_type]["AREA"]`
+- loop through the building area types: `for bat in list_building_area_types_and_zones:`
+	- if the bat is not equal to the predominant building area type: `if bat != predominant_building_area_type:`
+		- and if the area of this bat <= 20,000 ft2: `if list_building_area_types_and_zones[bat]["AREA"] <= 20000:`
+			- add this area to the area value for determining predominant HVAC system type: `area = area + list_building_area_types_and_zones[bat]["AREA"]`
```

Comment:
System types are determined based on area and number of floors of the entire building, not just area and number of floors of that BAT.

Reply:
yes, thank you.  updated